dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -936,6 +1140,93 @@ def test_json_decode_after_update(self):
 
         test_utils.assert_profiles_equal(deserialized, expected_profile)
 
+    def test_cms_heavy_hitters_threshold(self):
+
+        df_categorical = pd.Series(
+            [
+                "a",
+                "a",
+                "a",
+                "a",
+                "a",
+                "b",
+                "b",
+                "b",
+                "b",
+                "b",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+                "c",
+            ]
+        )
+        options = CategoricalOptions()
+        options.cms = True
+        options.cms_confidence = 0.95
+        options.cms_relative_error = 0.01
+        options.heavy_hitters_threshold = 2
+
+        profile = CategoricalColumn("test_name", options)
+        profile.update(df_categorical)
+
+        self.assertEqual(1, len(profile.categories))
```

Comment:
instead we should assert the dict itself for clarity.
I assume it would be {"c": 10}